deployer/README.md

```diff
@@ -92,6 +92,82 @@ yarn make-package
 and if the deployment package is different from what is already in AWS,
 it will upload and publish a new version.
 
+## Elasticsearch indexing
+
+You just need a URL (or host name) for an Elasticsearch server and the
+root of the build directory. The command will trawl all `index.json` files
+and extract all metadata and blocks of prose which gets HTML stripped.
```

Comment:
```suggestion
and extract all metadata and blocks of prose which get HTML stripped.
```

Reply:
I think it should probably have "its" inserted.
```suggestion
and extract all metadata and blocks of prose which gets its HTML stripped.
```
what do you think?

Reply:
Since it's plural (`blocks`), this reads better I think:
```suggestion
and extract all metadata and blocks of prose which get their HTML stripped.
```